test/e2e/setup/setup.go

```diff
@@ -115,3 +128,153 @@ func GeneratePasswordForUser(mdbu mdbv1.MongoDBUser, ctx *f.Context, namespace s
 
 	return password, f.Global.Client.Create(context.TODO(), &passwordSecret, &f.CleanupOptions{TestContext: ctx})
 }
+
+func deployOperator(c f.FrameworkClient) error {
```

Comment:
moved from testrunner/main.go

values now come from `testConfig` instead of `flags`. These are populated through env vars passed in `e2e.py`